IdentityCore/src/oauth2/MSIDTokenResponse.m

```diff
@@ -78,7 +78,10 @@ - (void)setIdToken:(NSString *)idToken
         
         NSError *localError;
         _idTokenObj = [self tokenClaimsFromRawIdToken:idToken error:&localError];
-        MSID_LOG_WITH_CTX_PII(MSIDLogLevelError, nil, @"Failed to init id token claims in %@, error: %@", self.class, MSID_PII_LOG_MASKABLE(localError));
+        if (localError)
```

Comment:
can we print it only if idToken is not-nil? I've also seen this error being printed when we don't even have an idToken.

Reply:
Sure